routing/routing_session.hpp

```diff
@@ -105,6 +106,15 @@ class RoutingSession
   inline void SetState(State state) { m_state = state; }
 
   Route const & GetRoute() const { return m_route; }
+  /// \returns true if any altitude information along |m_route| is available and
+  /// false otherwise.
+  bool HasRouteAltitude() const;
+  /// \brief copies route altitude information to |routeAltitudes| if any is available and
+  /// returns true. If no route altitude information is available returns false.
```

Comment:
So, why not write this:
```c++
If there's no navigation route, the method returns false.
```
in the comment? "If no route altitude information" sounds to generic and can be misinterpreted, as I did.